mobile/src/main/java/org/openhab/habdroid/ui/LogActivity.java

```diff
@@ -103,15 +109,16 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
     private class GetLogFromAdbTask extends AsyncTask<Boolean, Void, String> {
         @Override
-        protected String doInBackground(Boolean... clearBeforeRead) {
+        protected String doInBackground(Boolean... clear) {
             StringBuilder logBuilder = new StringBuilder();
             String separator = System.getProperty("line.separator");
             try {
-                if (clearBeforeRead[0]) {
+                if (clear[0]) {
```

Comment:
@maniac103 The doInBackground does two indipendent thing now. Should I split them in two AsyncTasks?

Reply:
No, I think it's fine as is. Chaining two threads won't gain us anything as dar as I can tell.